rflx/templates/rflx_arithmetic.ads

```diff
@@ -7,6 +7,8 @@ is
    type U64 is mod 2**64 with
      Annotate => (GNATprove, No_Wrap_Around);
 
+   type S63 is range 0 .. 2 ** 63 - 1;
```

Comment:
Is there a reason for keeping `U64`?

Reply:
It is still used in rflx-generic_types.ads/adb. I wouldn't want to change that entire unit.